hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/Event.java

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerts.api.model.event;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.hawkular.alerts.api.model.condition.ConditionEval;
+import org.hawkular.alerts.api.model.dampening.Dampening;
+import org.hawkular.alerts.api.model.trigger.Trigger;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+/**
+ * An Alert is an Event.  For the most part an Event can be thought of as an Alert without life-cycle. Alerts are
+ * always generated by a Trigger. Events may be generated by a Trigger or may be created directly via the API.
+ *
+ * @author Jay Shaughnessy
+ * @author Lucas Ponce
+ */
+public class Event {
+
+    @JsonInclude
+    protected String tenantId;
+
+    @JsonInclude
+    protected String id;
+
+    @JsonInclude
+    protected long ctime;
+
+    // A description of the event, suitable for display
+    @JsonInclude
+    private String eventText;
```

Comment:
Minor comment. Why not simply text ? Event.text looks more readable than Event.eventText.
Also Event.description looked fine but I see Event.text more generic.